IndoorAtlas/3.2.0-beta1/IndoorAtlas.podspec.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "IndoorAtlas",
+  "version": "3.2.0-beta1",
```

Comment:
Probably doesn't matter, but previously first beta has been just -beta

Reply:
Yes, this was @oseiskar suggestion to use beta1